docs/customizations/header.md

```diff
@@ -0,0 +1,301 @@
+headerTitle | ReactNode\|string | No | Used for overriding default header title | -
+headerMinimizeButton | ReactNode\|string | No | Used for overriding default minimize button | -
+headerCloseButton | ReactNode\|string | No | Used for overriding default close button | -
+
+### [IHeaderControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/header/interfaces/IHeaderControlProps.ts)
+
+| Attribute | Type | Required | Description | Default | 
+| - | - | - | - | - |
+| id     | string     | No | The top-level element id for the header | "oc-lcw-header"
+hideIcon | boolean | No | Whether to hide the icon on the header | false
+hideTitle | boolean | No | Whether to hide the title string on the header | false
+hideMinimizeButton | boolean | No | Whether to hide the minimize button on the header | false
+hideCloseButton | ReactNode\|string | No | Whether to hide the close button on the header | false
+onMinimizeClick | () => void | No | The callback function that will be triggered when the minimize button is clicked | [Minimizes the chat widget]
+onCloseClick | () => void | No | The callback function that will be triggered when the close button is clicked | [Closes the chat widget]
+minimizeButtonProps | [ICommandButtonControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/common/interfaces/ICommandButtonControlProps.ts) | No | Properties to further customize the default minimize button | [defaultHeaderControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/header/common/defaultProps/defaultHeaderControlProps.ts)
+closeButtonProps | [ICommandButtonControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/common/interfaces/ICommandButtonControlProps.ts) | No | Properties to further customize the default close button | [defaultHeaderControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/header/common/defaultProps/defaultHeaderControlProps.ts)
+headerIconProps | [IImageControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/common/interfaces/IImageControlProps.ts) | No | Properties to further customize the default header icon | [defaultHeaderControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/header/common/defaultProps/defaultHeaderControlProps.ts)
+headerTitleProps | [ILabelControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/common/interfaces/ILabelControlProps.ts) | No | Properties to further customize the default header title | [defaultHeaderControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/header/common/defaultProps/defaultHeaderControlProps.ts)
```

Comment:
my suggestion is to add permalinks to the exact code block where this and other properties are set

Reply:
I'm hesitant to add line links since it can change any time in future updates. Then we have to dive into each doc and change the line references. Instead, files are not likely to be removed/have reference changes

Reply:
I'm hesitant to add line links as they tend to change in the future. Then we have to dive into the docs and change every line reference